fem/bilininteg_ext.cpp

```diff
@@ -116,20 +115,20 @@ static void PADiffusionSetup2D(const int Q1D,
 {
    const int NQ = Q1D*Q1D;
    const DeviceVector W(w, NQ);
-   const DeviceTensor<4> J(j, 2, 2, NQ, NE);
-   DeviceTensor<3> y(op, 3, NQ, NE);
+   const DeviceTensor<4> J(j, NQ, 2, 2, NE);
```

Comment:
I think that ultimately we might have to introduce some `Layout` concept, but this would be premature to do it now.